src/JavaScript/SCORM_API_wrapper.js

```diff
@@ -369,7 +375,30 @@ pipwerks.SCORM.connection.terminate = function(){
 
         if(API){
 
-             if(scorm.handleExitMode && !exitStatus){
+            if(scorm.handleSessionTime){
+
+                var dtm = new Date(); 
+
+                //in the next line you substract the time recorded when initialising 
+                //the connection from the present time. 
+                var n = dtm.getTime() - dtmInitialized.getTime(); 
+                switch(scorm.version){ 
+
+                    //the time format is different on scorm 1.2 or 2004, so we use 
+                    //different conversions depending on the case 
+                    case "1.2" :
+                        //success = this.set("cmi.core.session_time",MillisecondsToCMIDuration(n)); 
+                        this.set("cmi.core.session_time",pipwerks.UTILS.MillisecondsToCMIDuration(n)); 
+                    break;
+                    case "2004":
+                        //success = this.set("cmi.session_time",centisecsToISODuration(Math.floor(n/10)));

```

Comment:
please remove commented-out line of code

Reply:
removed!